app/src/main/java/de/westnordost/streetcomplete/quests/QuestsModule.kt

```diff
@@ -525,4 +526,6 @@ fun questTypeRegistry(
 
     /* at the very last because it can be difficult to ascertain during day. used by OsmAnd if "Street lighting" is enabled. (Configure map, Map rendering, Details) */
     154 to AddWayLit(),
+
+    167 to AddParcelLockerBrand(),
```

Comment:
likely it can be put higher than at least some. Maybe below AddBikeParkingCapacity ?

Reply:
I'm going to add new quests related to parcel lockers in other PR's ^^

Reply:
That's cool! But please see

https://github.com/streetcomplete/StreetComplete/blob/8dbb9e664ad5c3b75a5ed1dddec94e0522940e7b/app/src/main/java/de/westnordost/streetcomplete/quests/QuestsModule.kt#L214-L244

TL;DR: The quests shouldn't be ordered by topic, but how easy it is to solve them.

Reply:
In other words, this quest can be put higher and should not be hidden by say maxspeed/lit quest (if these got enabled)

Reply:
I missed this line ^^. It has been modified